.dockerignore

```diff
@@ -0,0 +1,43 @@
+.attic/
+docs/
+tests/
+evals/
+etc
+src/aegis_ai_ml
+src/aegis_ai_chat
+
+
+# Environment
+.venv/
+env/
+venv/
+
+# IDE and OS
+.vscode/
+.idea/
+*.sublime-*
+Thumbs.db
+.DS_Store
+
+# Test and coverage
+.pytest_cache/
+.coverage
+htmlcov/
+.ruff_cache/
+*.pytest_cache*
+
+# Local data
+*.log
+
+# Packaging
+dist/
+build/
+*.egg-info/
+
+# Git
+.git
```

Comment:
@JimFuller-RedHat This breaks the CI because `Containerfile.eval` explicitly copies `.git` into the container image.  This is used to create optimized evaluation images that do not need to clone the repository at run time.  They only fetch the changes done since the image creation if needed.

Reply:
I have pushed commit 1ad4998abfdd26faa61628e48a9b3676080a9d23 to fix this.